src/sketchup_objects/hub.rb

```diff
@@ -205,9 +212,11 @@ def sensor?
     @is_sensor
   end
 
-  def attach_user(force)
+  def attach_user(force, name:)
```

Comment:
I guess the colon is not necessary?

Reply:
Now it is ```atach_user(weight: nil, name:)``` and then it makes sense, cause you can then set the name without setting the weight...

I just wanted to make sure it is named so you can set it single.